src/cmd/singularity/cli/build.go

```diff
@@ -101,7 +101,7 @@ var BuildCmd = &cobra.Command{
 					authWarning != auth.WarningTokenToolong &&
 					authWarning != auth.WarningTokenTooShort {
 					if authToken != "" {
-						b = build.NewRemoteBuilder(args[0], def, false, remoteURL, authToken)
+						b = build.NewRemoteBuilder(args[0], "", def, false, remoteURL, authToken)
```

Comment:
Should this be a flag? `--libraryurl` pointing to the default lib URL?

Reply:
Trying to keep it simple for now. I think we can expose that if/when necessary.